repose-aggregator/tests/performance-tests/Jenkinsfile

```diff
@@ -27,14 +27,17 @@ stage("Performance Test") {
     def performanceJenkinsJob = "performance-test"
     def performanceTestParam = "perf_test"
     def extraOptionsParam = "extra_options"
+    def random = new Random()
 
     def jobsToRun = [:]
 
     for (int index = 0; index < perfTests.size(); index++) {
         def perfTest = perfTests[index]
         jobsToRun[perfTest] = {
             retry(3) {
-                build(job: performanceJenkinsJob, parameters: [
+                build(job: performanceJenkinsJob,
+                      quietPeriod: (random.nextInt() % 585) + 15,
```

Comment:
I would almost suggest giving the magic numbers named variables, but the behavior is pretty obvious, and it's not like we are likely to modify this file frequently. I would take it either way.